### PR TITLE
PF-2610: Validate policyInputs during PAO Create/Replace

### PR DESCRIPTION
**Pull request by mikenev**
We had validation while updating & merging policies, but not during initial create or replace. We should validate those too.

### CODE REVIEW

File: service/src/main/java/bio/terra/policy/service/policy/PolicyUnknown.java [hunk 1]
```diff
@@ -72,13 +72,13 @@ public PolicyInput remove(PolicyInput target, PolicyInput removePolicy) {
   }
 
   /**
-   * Unknown policies will be seen as invalid.
+   * Unknown policies will be seen as valid.
```
Comment (zloery) [inline service/src/main/java/bio/terra/policy/service/policy/PolicyUnknown.java:75]: Why this change in behavior?

**Review by zloery — APPROVED** [review]
LGTM, one question about unknown policies.

Let's also keep an eye out on WSM/CLI tests for any unexpected failures as this goes live

File: service/src/main/java/bio/terra/policy/service/policy/PolicyUnknown.java (hunk 1 above)
Comment (mikenev) [reply service/src/main/java/bio/terra/policy/service/policy/PolicyUnknown.java:75]: I should have put that in the description :/ This came up because we set these to invalid in an earlier PR, but didn't check that with Product. After more discussion with Product, it makes most sense to require validation for Terra namespace policies, but be flexible with other namespaces.
